### DISCUSSION TITLE
claude3已发布api,能力超越GPT4.0,能否增加这个接口调用

### DISCUSSION THREAD

**Discussion by KaiTraders** [immersive-translate/immersive-translate#1491] [Ideas]
### 新功能 | New Feature

1.claude3已发布api,能力超越GPT4.0,能否增加这个接口调用
2.
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.claude3已发布api,能力超越GPT4.0,能否增加这个接口调用
2.
3.

**Comment by theowenyoung** [comment]
好的～研究一下，多些反馈。
